src/components/earn/PoolCard.tsx

```diff
@@ -113,7 +127,14 @@ export const PoolCard: React.FC<Props> = ({ compoundBotSummary }: Props) => {
   const userAprMode = useIsAprMode()
   const { address } = useContractKit()
   const dispatch = useDispatch()
+
   const { v2Trade: trade, inputError: swapInputError } = useDerivedSwapInfo()
+  const { priceImpactWithoutFee } = trade ? computeTradePriceBreakdown(trade) : {}
+  const isPriceImpactTooHigh = priceImpactWithoutFee
+    ? priceImpactWithoutFee.lessThan(BLOCKED_PRICE_IMPACT_NON_EXPERT)
+      ? false
+      : true
+    : false
```

Comment:
```suggestion
  const isPriceImpactTooHigh = priceImpactWithoutFee?.greaterThan(BLOCKED_PRICE_IMPACT_NON_EXPERT)
```
The double ternary is very confusing. I think this is logically equivalent.

Reply:
you could add a `!!` prefix if you are worried about it being undefined

Reply:
If it's undefined, we'll get a TypeError when attempting to call `lessThan` or `greaterThan` on it though, right?

Reply:
I definitely agree that the double ternary is confusing, though.

Reply:
I think the question mark before the period will prevent the TypeError

Reply:
oh you're totally right, ty!